test/integ_tests/conftest.py

```diff
@@ -162,15 +162,23 @@ def job_failed_name(request):
 
 @pytest.fixture(scope="session", autouse=True)
 def completed_quantum_job(aws_session, job_completed_name):
-    account = boto3.client("sts").get_caller_identity().get("Account")
-    region = aws_session.region
-    job = AwsQuantumJob(arn=f"arn:aws:braket:{region}:{account}:job/{job_completed_name}")
+    job_arn = [
+        job["jobArn"]
+        for job in boto3.client("braket").search_jobs(filters=[])["jobs"]
+        if job["jobName"] == job_completed_name
+    ][0]
```

Comment:
Why not put job name in filter instead of `if job["jobName"] == job_completed_name`?
```suggestion
    job_arn = boto3.client("braket").search_jobs(
        filters=[{"name": job_completed_name}]
    )["jobs"][0]["jobArn"]
```